SampleGetApi/Tests/FactSampleTest.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using AventStack.ExtentReports;
+using AventStack.ExtentReports.MarkupUtils;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+using Xunit.Abstractions;
+
+namespace SampleGetApi.Tests
+{
+    // Collection attribute to group related tests
+    [Collection("SampleApi")]
+    public class FactSampleTest : BaseClass
+    {
+        // Public reference to RestClient
+        public RestClient restClient;
+
+        // Constructor to initialize RestClient at the start of the scripts
+        public FactSampleTest(ITestOutputHelper output) : base(output)
+        {
+            // Creating a reference to RestClient and setting the Base Url
+            restClient = new RestClient(new RestClientOptions
+            {
+                BaseUrl = new Uri("https://api.publicapis.org/")
+            });
+
+            // Logging test status information
+            _test?.Log(Status.Info, "RestClient initialized with base URL: https://api.publicapis.org/");
+        }
+
+        [Fact]
+        public void GetEntries()
```

Comment:
Can we do better test naming. 
with method, input and expected?

Reply:
Noted